src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
I guess I should take a step back and ask why users would want to specify
index ordering. Normally it would be to optimize the final ordering of
results, but since we don't support ORDER BY anyway, this isn't (currently)
relevant. Even if we did support ORDER BY, an extra sort operator at the
root probably isn't important enough for a user to try to optimize away
with clever indexes. So I'm not sure exactly what I was thinking when I
proposed this. Can you think of a compelling use case for user-specified
index ordering? Would it help us do well on any benchmarks?

On Tue, Jul 12, 2016 at 12:02 AM, jortiz16 <notifications@github.com> wrote:

> In src/edu/washington/escience/myria/coordinator/MasterCatalog.java
> <https://github.com/uwescience/myria/pull/830#discussion_r70386203>:
>
> > @@ -113,6 +114,7 @@
> >      + "    col_index INTEGER NOT NULL,\n"
> >      + "    col_name TEXT,\n"
> >      + "    col_type TEXT NOT NULL,\n"
> > +    + "    is_indexed INTEGER NOT NULL, \n"
>
> So I basically relied on createIndexes method that already existed in the
> JdbcAccessMethod class. These methods did not have the ability to specify
> index order. Should I add this feature to the createIndex methods and let
> users specify sorting order through the createIndex encoding? I don't think
> this would be difficult to do....Although I'm not yet sure how users should
> specify this order
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/uwescience/myria/pull/830/files/4f8f7d56457c12dc57f9f0975fc979c49d08ae0e#r70386203>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIBYSioQ_Exs53h81FCGNmHQdeecobNks5qUzwHgaJpZM4If5_N>
> .
>